crane/template.go

```diff
@@ -0,0 +1,105 @@
+package crane
+
+import (
+	"fmt"
+	"github.com/michaelsauter/crane/print"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+	"text/template"
+)
+
+type TemplateData struct {
+	Graph              DependencyGraph
+	TargetedContainers Containers
+}
+type TemplateDataForContainer struct {
+	Graph              DependencyGraph
```

Comment:
might be worth considering exposing `Dependencies` as an interface rather than a struct for the sake of abstraction